test/SoulboundCreditScore.test.ts

```diff
@@ -54,7 +54,7 @@ const signMintCreditScoreToIdentity = async (
     },
     // Types
     {
-      MintCreditScore: [
+      Mint: [
```

Comment:
his will create a breaking change, the SDK uses MintCreditScore for all the credit scores